ui/pages/reservation-unit/[...params].tsx

```diff
@@ -376,23 +376,27 @@ const ReservationUnitReservationWithReservationProp = ({
   });
 
   const { pk: reservationPk } = reservation || {};
+  const sortedAgeGroups = ageGroups.sort((a, b) => a.minimum - b.minimum);
 
   const options = useMemo(
     () => ({
       purpose: reservationPurposes.map((purpose) => ({
         label: getTranslation(purpose, "name"),
         value: purpose.pk,
       })),
-      ageGroup: ageGroups.map((ageGroup) => ({
-        label: `${ageGroup.minimum} - ${ageGroup.maximum}`,
-        value: ageGroup.pk,
-      })),
+      // the sortedAgeGroups array has "1 - 99" as the first element, so let's move it to the end for correct order
+      ageGroup: [...sortedAgeGroups.slice(1), sortedAgeGroups[0]].map(
```

Comment:
Not sure if it's guarantied anywhere that ageGroups.length > 0.

So I'd prefer [...arr.slice(1), ...arr.slice(0, 1)] or other null checks because
```
const arr = []
[arr.slice(1), arr[0]]  === [undefined]
[...arr.slice(1), ...arr.slice(0, 1)] === []
```

Reply:
I added a null check, which gives a `console.warn()` if there are no ageGroups, and cleaned up the array construction as per your example.